.github/dependabot.yml

```diff
@@ -4,12 +4,6 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
-  # duplicated gomod checks for each module pending:
-  # https://github.com/dependabot/dependabot-core/issues/6012
-  - package-ecosystem: "gomod"
-    directory: "/tools/internal/gen"
-    schedule:
-      interval: "weekly"
```

Comment:
Not sure if this ever did anything - there is only a go.mod file in /tools and nothing in the gen folder.